spring-boot-autoconfigure/pom.xml

```diff
@@ -497,5 +497,11 @@
 			<artifactId>slf4j-jdk14</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.cxf</groupId>
+			<artifactId>cxf-bundle</artifactId>
+			<version>3.0.0-milestone2</version>
```

Comment:
I have incorporated all changes yet to check in. But only thing that mismatch was cxf bundle -2.7.16 is stable release, which is using the spring 3.0.7 release, which is failing and not supported by spring boot current version. CXF bundle, 3.0.0-milestone2 is using spring 3.2.6.release. it is working without any issue with spring boot. But dont see any release plan for cxf bundle-3.0.0. I am stuck there. Please guide me here, can we go with the CXF bundle, with 3.0.0-milestone2 version here.